src/internal/connector/onedrive/collection.go

```diff
@@ -418,11 +418,13 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 					if err != nil {
 						if item.GetMalware() != nil {
 							logger.Ctx(ctx).With("error", err.Error(), "malware", true).Error("downloading item")
+							// el.AddSkip(fault.FileSkip(fault.SkipMalware, itemID, itemName, ptr.Val(pr.GetId()), ptr.Val(pr.GetName())))
+							el.AddSkip(fault.FileSkip(fault.SkipMalware, itemID, itemName, "container-id", "container-name"))
 						} else {
 							logger.Ctx(ctx).With("error", err.Error()).Error("downloading item")
+							el.AddRecoverable(clues.Stack(err).WithClues(ctx).Label(fault.LabelForceNoBackupCreation))
 						}
 
-						el.AddRecoverable(clues.Stack(err).WithClues(ctx).Label(fault.LabelForceNoBackupCreation))
 						return nil, err
```

Comment:
Not in this case, because we're inside the lazy reader.  An error will communicate to kopia that the reader has failed.  I added a comment in the code to make that clear.